src/api/app/models/package.rb

```diff
@@ -309,23 +308,21 @@ def check_write_access!(ignoreLock = nil)
     end
   end
 
-  def check_devel_packages
-    if develpackages.any?
-      develpackages.each do |package|
-        # We only care about packages in foreign projects
-        if package.project.name != self.project.name
-          self.errors.add(:base, "used as devel package by #{package.project.name}/#{package.name}")
-        end
+  def check_weak_dependencies(ignore_local = false)
```

Comment:
hm, yes, my plan was to bring the Project.can_be_really_deleted? method in sync .... but we can as well skip it....